modules/gateway/rpc.go

```diff
@@ -114,6 +114,7 @@ func (g *Gateway) threadedHandleConn(conn modules.PeerConn) {
 		g.log.Printf("WARN: incoming conn %v requested unknown RPC \"%v\"", conn.RemoteAddr(), id)
 		return
 	}
+	g.log.Printf("INFO: incoming conn %v requested RPC \"%v\"", conn.RemoteAddr(), id)
```

Comment:
if you put this here the gateway.log is going to be too big. It's already gigabytes large for some longer-running nodes. this log statement can't go in a normal log (Luke likes the idea of a rolling log, though I'm not sure that's the best approach - it's probably good enough in this case to just not log at all)